### PR TITLE
implement Discord login

### PR DESCRIPTION
**Pull request by ghost**
This needs a Discord application created (at <https://discord.com/developers/applications>)

### CODE REVIEW

**Comment by goldyydev** [comment]
I think this is something for @benawad to take a look at due to it's sheer gravity on the app.

**Comment by benawad** [comment]
Code looks good, I will test this out locally later

**Comment by yummyweb** [comment]
I like this feature. It would be pretty cool.

**Comment by overlisted** [comment]
did you also add the login with discord text to all langauges lol

**Comment by ghost** [comment]
> did you also add the login with discord text to all langauges lol

`yarn i18` did that

**Comment by ghost** [comment]
pinging @ityonemo as requested from the discord server

File: kofta/public/locales/de/translation.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
       "featureText_6": "Angetrieben durch Doge",
       "loginGithub": "mit GitHub anmelden",
       "loginTwitter": "mit Twitter anmelden",
-      "createTestUser": "Testnutzer erstellen"
+      "createTestUser": "Testnutzer erstellen",
+      "loginDiscord": "login with Discord"
```
Comment (gruselhaus) [inline kofta/public/locales/de/translation.json:58]:
```suggestion
      "loginDiscord": "mit Discord anmelden"
```

**Review by gruselhaus — CHANGES_REQUESTED** [review]
nvm.

File: kofta/public/locales/de/translation.json (hunk 1 above)
Comment (gruselhaus) [inline kofta/public/locales/de/translation.json:63]:
```suggestion
      "createTestUser": "Testbenutzer erstellen",
      "loginDiscord": "mit Discord anmelden"
```

**Review by gruselhaus — CHANGES_REQUESTED** [review]
This should be changed.

File: kofta/public/locales/de/translation.json (hunk 1 above)
Comment (overlisted) [reply kofta/public/locales/de/translation.json:58]: we shouldn't include i18n in this PR

**Comment by overlisted** [comment]
are we waiting for something

**Comment by ityonemo** [comment]
sorry, it took me so long to see this but gmail suddenly flipped to sinking all of my github stuff to spam folder =( Elixir bits look fine to me.

**Comment by HarrisonMayotte** [comment]
@overlisted I believe we're waiting for a review / discord application to be made by Ben.

**Comment by ityonemo** [comment]
might also need a merge resolution.  there's been a lot of refactoring going on underfoot, sorry not sorry

**Comment by amitojsingh366** [comment]
what's the status on this pr?

**Comment by gruselhaus** [comment]
> might also need a merge resolution. there's been a lot of refactoring going on underfoot, sorry not sorry

**Comment by overlisted** [comment]
hopefully i didn't mess this up

**Comment by overlisted** [comment]
oops i did

**Comment by TheOtterlord** [comment]
I've added the `help-wanted` label as we are awaiting help with the merge conflic resolution

**Pull request merged**